src/main/webapp/query-control.js

```diff
@@ -109,13 +109,21 @@ function resetHistoryList() {
 
 function toggleHistory(shouldShow) {
```

Comment:
Maybe rename this function to toggleLoggedInContent or something that indicates it's toggling all the things that the person has to be logged in to see?

Reply:
changed to toggleUserContent()